src/com/cft/contactmerge/io/tests/TsvImporterTest.java

```diff
@@ -71,9 +71,9 @@ void Load_EmptyStream() throws IOException {
      * Testing Iterator
      * -------------------------------------------------------------------------------- */
 
-    private static String testTsv =  "Bidder #,CRM ID,Type,Salutation,First,Last,Company,Phone #,E-mail,Address,City,State,Zip,Country,User Defined 1,User Defined 2,Notes,VIP,Table #,Table Name,Ticket #,Ticket Package,Express Checkout Credit Card,RSVP Status,Checked In\n" +
-            "5,,Person,,John,Doe,,(520) 123-4567,jdoe@gmail.com,123 Main St,Tucson,Arizona,85716,,,Beider,,No,18,Group 1,3,Comp Ticket-Honoree,,,Yes\n" +
-            "5,,Person,,Adam,Smith,,(520) 111-2222,adam.smith@yahoo.com,1010 Speedway Blvd,Tucson,AZ,85716,,,,,No,18,Group 1,1,Comp Ticket-Honoree,,,Yes\n";
+    private static String testTsv =  "Bidder #	CRM ID	Type	Salutation	First	Last	Company	Phone #	E-mail	Address	City	State	Zip	Country	User Defined 1	User Defined 2	Notes	VIP	Table #	Table Name	Ticket #	Ticket Package	Express Checkout Credit Card	RSVP Status	Checked In\n" +
+            "5		Person		John	Doe		(520) 123-4567	jdoe@gmail.com	123 Main St	Tucson	Arizona	85716			Beider		No	18	Group 1	3	Comp Ticket-Honoree			Yes\n" +
```

Comment:
I think it will be more readable if we have the escape code for tab which is "\t".

Reply:
Will do.